scripts/setup-k8s-master.sh.in

```diff
@@ -120,13 +140,10 @@ kubectl apply -f {{NetworkPolicyUrl}}
 
 # Use kubeadm's kubeconfig command to grab a client-cert-authenticated
 # kubeconfig file for administrative access to the cluster.
-# Because kubeadm doesn't set controlPlaneEndpoint as master, the
-# --apiserver-advertise-address is set to the load balancer external ipv4 address.
```

Comment:
This comment just didnt seem accurate; we do set the controlPlaneEndpoint, at least in the config. As a result I removed this flag. Everything still worked as expected.

Reply:
Was wrong here; added back a more clear comment based on feedback.